yocto-dev/meta-custom/conf/layer.conf

```diff
@@ -0,0 +1,13 @@
+# We have a conf and classes directory, add to BBPATH
```

Comment:
This comment is not useful. Please add more context.

Reply:
This entire file is auto-generated when creating a layer from scratch using `bitbake-layers` functionality. Not sure whether it's still worth commenting.

Reply:
See above comment about `yocto-dev/build_conf/local.conf`. If this is not the same situation, then add a one line readme to this directory stating what what this file is and that it is auto-generated.